lib/easypost/report.rb

```diff
@@ -5,13 +5,42 @@ class EasyPost::Report < EasyPost::Resource
   # Create a Report.
   def self.create(params = {}, api_key = nil)
     url = "#{self.url}/#{params[:type]}"
+
+    columns = params[:columns] || []
+    additional_columns = params[:additional_columns] || []
+
+    url += '?'
+    columns.each do |column|
+      url += "columns[]=#{column}&"
+    end
+    additional_columns.each do |additional_column|
+      url += "additional_columns[]=#{additional_column}&"
+    end
+
+    params = params.reject { |k, _| [:columns, :additional_columns].include?(k) }
```

Comment:
Smart! Can we add a comment here why we're pulling these out? Something like `already included in the URL params` could do

Reply:
Resolved in 838f54d0dbb44f5b78595d4a0840983b7d997834